client/csharp/Script Wars Client/Script Wars Client/ScriptWars/Games/Tanks/Coordinates.cs

```diff
@@ -17,7 +17,7 @@ public class Coordinates
         /// <param name="y">Y coordinate.</param>
         public Coordinates(int x, int y)
         {
-            X = y;
+            X = x;
```

Comment:
Wow I'm an idiot. Didn't even consider that I screwed up that constructor instead of something reading into it (which was more likely to be error prone since the read order matters).